modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -144,36 +234,284 @@ function inope_policy_review_form($form, &$form_state, $node) {
 
 /**
  * The iNope Policy Review Form submit handler.
+ *
+ * This submit handler is broken up into 5 parts.
+ * PART 1 - Set variables and load necessary objects.
+ * PART 2 - Update existing entries if policy review id is a non empty string.
+ * PART 3 - Insert an approved response.
+ * PART 4 - Insert a changes requested response.
+ * PART 5 - Catch errors.
  */
 function inope_policy_review_form_submit($form, &$form_state) {
-  // Load objects from which their properties will be displayed.
+  // PART 1 - Load objects from which their properties will be displayed.
   $account = user_load($form_state['values']['inope_policy_review_hidden_uid']);
   $node = node_load($form_state['values']['inope_policy_review_hidden_nid']);
+  $text = check_plain($form_state['values']['inope_policy_review_changes_requested_textarea']);
 
-  // If 'I have reviewed this policy and approved it for use' is selected.
-  if ($form_state['values']['inope_policy_review_status_radio'] == 0) {
-    drupal_set_message(t('Policy approved for use'));
-    // TODO: put this into the database.
-  }
-  // If 'I have reviewed this policy and would like to see changes' is selected.
-  elseif ($form_state['values']['inope_policy_review_status_radio'] == 1) {
+  // PART 2 - Check to see if we are updating an existing entry by checking if
+  // the form value hidden_policy_review_id is set to a non empty string.
+  if ($form_state['values']['inope_policy_review_hidden_policy_review_id'] != '') {
+
+    // Run the update query using form_state values.
+    inope_policy_review_update_query($form_state);
+
+    // Clear previous messages and thank the user for their update.
+    drupal_get_messages();
     drupal_set_message(
-      t('User : @userid : @username submitted the following change request:', array(
-        '@userid' => $account->uid,
+      t('Thank you @username for your update to the policy', array(
         '@username' => $account->name,
```

Comment:
As long as those values aren't used in the database -- that is global $user will only get the logged in user, not the user who's response is being edited.  Do you need both actually?  I don't have time to look at the full diff -- but if you are using the user id anywhere besides just the message you probably want it to be sensitive to the response and not just who is logged in editing it.

I wish the editing thing had been more clear in the problem, its not really what I had hoped for and makes this problem more complex that it was.  Oh well, atleast it is good to see your thinking on it.